### PR TITLE
common serde helpers for arrays

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
cc @Emilgardis

File: svd-rs/src/lib.rs [hunk 1]
```diff
@@ -238,3 +238,21 @@ impl<T> EmptyToNone for Option<Vec<T>> {
         self.and_then(|v| if v.is_empty() { None } else { Some(v) })
     }
 }
+
+#[cfg(feature = "serde")]
+#[derive(serde::Serialize)]
+struct SerArray<'a, T> {
+    #[cfg_attr(feature = "serde", serde(flatten))]
```
Comment (Emilgardis) [inline svd-rs/src/lib.rs:245]: These cfg_attrs are useless, since the struct is only output with the serde feature enabled

**Review by Emilgardis — CHANGES_REQUESTED** [review]

File: svd-rs/src/lib.rs (hunk 1 above)
Comment (burrbull) [reply svd-rs/src/lib.rs:245]: good catch

Comment (burrbull) [reply svd-rs/src/lib.rs]: fixed

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**